test/e2e/app/snapshots.go

```diff
@@ -88,11 +87,10 @@ func (s *SnapshotStore) Create(state *State) (abci.Snapshot, error) {
 	if err != nil {
```

Comment:
Is `state.Export()` deterministic?

Reply:
probably not necessarily.

Reply:
I believe that the json encoder uses the in-memory order of the map, so it should be stable in the short term, depending on how the map is modified. As far as I know, if this were a problem, we'd see hash mis-matches, and that's not a problem that's happening (one of my changes is to sort the keys before we hash them, so I think that might solve the problem.) 

I think the changes to fix this are potentially pretty intensive, unless we have a canned ordered-map-serialization method.

Reply:
Ok. If it hasn't really been a problem before then I think it'll be alright as it is